src/views/Project/index.jsx

```diff
@@ -0,0 +1,130 @@
+import { hot } from 'react-hot-loader';
+import { Component, Fragment } from 'react';
+import Typography from '@material-ui/core/Typography';
+import projects from '../../data/loader';
+import TasksTable from '../../components/TasksTable';
+
+@hot(module)
+export default class Project extends Component {
+  state = {
+    fileName: this.props.match.params.project,
+    projects,
+    data: [
+      {
```

Comment:
We shouldn't put `data` in `state` because it's not changing. Define it on top:

```jsx
const data = [
// ...
];

@hot(module)
export default class Project extends Component {
  // ...
}
```